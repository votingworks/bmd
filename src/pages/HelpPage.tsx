import React from 'react'

import ButtonBar from '../components/ButtonBar'
import LinkButton from '../components/LinkButton'
import Main, { MainChild } from '../components/Main'
import Prose from '../components/Prose'

const HelpPage = () => {
  return (
    <>
      <Main>
        <MainChild>
          <Prose>
            <h1>Help</h1>
            <p>Help content will be available here.</p>
          </Prose>
        </MainChild>
      </Main>
      <ButtonBar secondary>
        <LinkButton goBack>Back</LinkButton>
        <div />
      </ButtonBar>
    </>
  )
}

export default HelpPage
