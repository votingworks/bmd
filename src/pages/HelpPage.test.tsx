import { axe } from 'jest-axe'
import React from 'react'
import { Route } from 'react-router-dom'

import { render } from '../../test/testUtils'

import HelpPage from './HelpPage'

it(`renders HelpPage`, async () => {
  const { container } = render(<Route path="/" component={HelpPage} />, {
    route: '/',
  })
  expect(container.firstChild).toMatchSnapshot()
  expect(await axe(container.innerHTML)).toHaveNoViolations()
})
