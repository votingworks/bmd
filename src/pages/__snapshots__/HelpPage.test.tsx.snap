// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders HelpPage 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  overflow: auto;
  padding: 1rem 0.5rem 2rem;
}

.c1 {
  margin: 0 0;
  max-width: 35rem;
}

.c2 {
  width: 100%;
  max-width: 66ch;
  line-height: 1.2;
}

.c2 h1 {
  margin: 2rem 0 1rem;
  line-height: 1.1;
  font-size: 1.5rem;
}

.c2 h2 {
  margin: 1.5rem 0 0.75rem;
  font-size: 1.25rem;
}

.c2 h3,
.c2 p {
  margin-top: 1rem;
  margin-bottom: 1rem;
  font-size: 1rem;
}

.c2 h1 + h2 {
  margin-top: -0.75rem;
}

.c2 h1 + p,
.c2 h2 + p {
  margin-top: -0.75rem;
}

.c2 h3 + p {
  margin-top: -1rem;
}

.c2 > :first-child {
  margin-top: 0;
}

.c2 > :last-child {
  margin-bottom: 0;
}

@media (min-width:480px) {
  .c0 {
    padding: 1rem 1rem 2rem;
  }
}

@media print {
  .c0 {
    -webkit-box-pack: start;
    -webkit-justify-content: flex-start;
    -ms-flex-pack: start;
    justify-content: flex-start;
    padding: 0;
  }
}

@media print {
  .c1 {
    margin: 0;
    max-width: 100%;
  }
}

@media (min-width:480px) {
  .c2 {
    line-height: 1.3;
  }
}

<main
  class="c0"
>
  <div
    class="c1"
  >
    <div
      class="c2"
    >
      <h1>
        Help
      </h1>
      <p>
        Help content will be available here.
      </p>
    </div>
  </div>
</main>
`;
